index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+  <body>
+      <header>
+        <div class="toolbar">
+            <div>
+                <h1>Ishmael</h1>
+            </div>
+            <div>
+                <img src="img/union.png" alt="">
+            </div>
+        </div>
+        <div class="headline">
+            <div>
+                <h2>Hey There. I’m Alex</h2>
```

Comment:
- [ ] Again great job here 👍 However as mentioned in a previous comment this part should have the `<h1>` tag.